ondewo/nlu/agent.proto

```diff
@@ -628,6 +638,33 @@ message ExportAgentResponse {
     }
 }
 
+message ExportBenchmarkAgentRequest {
+    // Agent related info
+    // Required. The project that the agent to export is associated with.
+    // Format: `projects/<Project ID>`.
+    string parent = 1;
+    // Optional. The compression level of the zip file created.
+    // Valid range 1-9; the grpc-default value 0 will be mapped to 1,
+    // other values cause an error in the request validation handler.
+    int32 compression_level = 2;
+
+    // Train-test split related parameters, for further info,
+    // check https://scikit-learn.org/stable/modules/generated/sklearn.model_selection.train_test_split.html
+    // Optional: bigger than zero
+    float test_size = 3;
+    // Optional: bigger than zero
+    float train_size = 4;
+    // Optional: random seed
+    int32 random_state = 5;
+    // Always True
```

Comment:
The field is ignored in the back-end since the stratified split always needs to shuffle.
Therefore, this actually cannot be configured.